src/data/equiv/basic.lean

```diff
@@ -129,6 +129,9 @@ def simps.inv_fun (e : α ≃ β) : β → α := e.symm
 
 initialize_simps_projections equiv (to_fun → apply, inv_fun → symm_apply)
 
+-- Generate the `simps` projections for previously defined equivs.
```

Comment:
I can also split the changes in this file into a new PR if you prefer. (Overall there is a big defecit of `simps`es in `data/equiv`.)